Fantome.League/IO/BIN/BINFileValueList.cs

```diff
@@ -24,7 +24,7 @@ public class BINFileValueList
         /// <summary>
         /// A Collection of <see cref="BINFileValue"/>
         /// </summary>
-        public List<BINFileValue> Entries { get; private set; } = new List<BINFileValue>();
+        public List<object> Entries { get; private set; } = new List<object>();

```

Comment:
I don't like that "object" thing. If I understand correctly Entries can either be BINFileValue or BINFileValuePair, so why not create an interface/abstract class?

Reply:
I actually though about that aswell, I will think about it